### PR TITLE
PMM-6215 RN 2.9.0

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**

### CODE REVIEW

File: source/conf.py [hunk 1]
```diff
@@ -258,7 +258,7 @@
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [
-  ('index', 'PerconaMonitoringAndManagement.tex', u'Percona Monitoring and Management Documentation',
+  ('index', 'PerconaMonitoringAndManagement-%s.tex' % release, u'Percona Monitoring and Management Documentation',
```
Comment (PaulJacobs-percona) [inline source/conf.py:261]: Adds a version suffix to the PDF so I don't have to prior to sending PDF to marketing.

File: source/release-notes/2.9.0.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL
+performance.
+
+New Features
+================================================================================
+
+* :jirabug:`PMM-6124`: New MongoDB Replica Set and Cluster Summary dashboards
+* :jirabug:`PMM-4192`: JSON format for MongoDB EXPLAIN
+* :jirabug:`PMM-5604`: User interface for MongoDB EXPLAIN
+* :jirabug:`PMM-5563`: Per-Service and per-Node Annotations (This completes the work on improvements to the Annotation functionality.)
```
[the remark this answers is not in the document]
Comment (PaulJacobs-percona) [reply source/release-notes/2.9.0.rst:20]: Jirabot doesn't report epics only their constituents.

File: source/release-notes/2.9.0.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+:Installation: `Installing Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/2.x/install/index-server.html>`_
+
+*PMM* (`Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/index.html>`_)
+is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL
+performance.
+
+New Features
+================================================================================
+
+* :jirabug:`PMM-6124`: New MongoDB Replica Set and Cluster Summary dashboards
```
[the remark this answers is not in the document]
Comment (PaulJacobs-percona) [reply source/release-notes/2.9.0.rst:17]: I am relying on jirabot to churn out the texts. I do not change issue parameters to alter their inclusion, only reword their summaries.

**Comment by PaulJacobs-percona** [comment]
> @PaulJacobs-percona , there are still no QAN-migration related epics which should be added as we discussed: https://jira.percona.com/browse/PMM-5516, https://jira.percona.com/browse/PMM-5125

Also as we discussed, you need to discuss with @peterschwaller how to get epics included.

File: source/release-notes/2.9.0.rst [hunk 4]
```diff
@@ -11,11 +11,16 @@
 is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL
 performance.
 
+Highlights
+==========
+
+This release brings a major rework of the Query Analytics (QAN) component, completing the migration from Angular to React, and adding new UI functionality and features. You can read more in the accompanying blog post (`here < https://www.percona.com/blog/2020/07/13/qan-release-post/>`_).
```
Comment (alexanderblinkov-percona) [inline source/release-notes/2.9.0.rst]: @PaulJacobs-percona , could you also add here the links to QAN epics ( https://jira.percona.com/browse/PMM-5516, https://jira.percona.com/browse/PMM-5125 ) as we discussed.

Comment (PaulJacobs-percona) [reply source/release-notes/2.9.0.rst]: Ok will test they work with :jirabug: and add them.

**Pull request merged**